infra/base-images/base-builder/Dockerfile

```diff
@@ -59,7 +59,14 @@ RUN export PYTHON_DEPS="\
     apt-get remove -y $PYTHON_DEPS  # https://github.com/google/oss-fuzz/issues/3888
 
 # Install latest atheris for python fuzzing, pyinstaller for fuzzer packaging.
-RUN unset CFLAGS CXXFLAGS && pip3 install -v atheris pyinstaller==4.1
+RUN unset CFLAGS CXXFLAGS && \
```

Comment:
Does atheris use a different libFuzzer version than the rest of our fuzz targets?